### DISCUSSION TITLE
Add System Message

### DISCUSSION THREAD

**Discussion by JTejero30** [JoshuaC215/agent-service-toolkit#95] [Ideas]
Hi!
First of all congratulations for this project!
I’m trying to add a System message to customize and extend the agent’s functionality, but I’m not sure where it should go in the code. If anyone can guide me or provide examples, I’d greatly appreciate your help!

**Comment by madtank** [comment]
Look under src/agents, there is research_assistant.py that has the graph that includes the agent with system prompt.

**Comment by JoshuaC215** [comment]
Thanks for the Q and responding so quickly @madtank ! [here's the mentioned section](https://github.com/JoshuaC215/agent-service-toolkit/blob/cb450b5c20fbbd7a0e0212e5d1da639639e2c308/src/agents/research_assistant.py#L37-L58).

**Comment by JTejero30** [comment]
Thank u both @madtank @JoshuaC215  !!
